src/parser/creative_linear_parser.js

```diff
@@ -158,6 +159,28 @@ export function parseCreativeLinear(creativeElement, creativeAttributes) {
 
           creative.mediaFiles.push(mediaFile);
         });
+
+      const mezzanineElement = parserUtils.childByName(mediaFilesElement, 'Mezzanine');
+      if (mezzanineElement) {
+        const mezzanine = new Mezzanine();
+        mezzanine.id = mezzanineElement.getAttribute('id');
+        mezzanine.fileURL = parserUtils.parseNodeText(mezzanineElement);
+        mezzanine.deliveryType = mezzanineElement.getAttribute('delivery');
+        mezzanine.codec = mezzanineElement.getAttribute('codec');
+        mezzanine.mimeType = mezzanineElement.getAttribute('type');
+        mezzanine.width = parseInt(
+            mezzanineElement.getAttribute('width') || 0
```

Comment:
As it's a mandotory field it should raise an error if width / height is not defined

Reply:
And `parseInt` must always have the second parameters to specify decimal base, otherwise it will try to guess it and can create some bugs.

Reply:
> As it's a mandotory field it should raise an error if width / height is not defined

It's ok not to crash the whole library on the minor issue. VAST is a standard, but not all agencies strictly follow the standards unfortunately.

Reply:
Yes of course it shouldn't crash but if mandatory field are not provided this mezzanine should be ignored. Because lot of agencies don't respect the standard, vast-client have to respect it. I agree to not crash everything but if someone else open a PR saying my agencies want default value at 1280/720 because our ads are all in HD what should we do ? That's why we try to be as close as to the IAB specification.

Reply:
valid point